src/app/modules/group/components/group-invite-users/group-invite-users.component.html

```diff
@@ -1,7 +1,7 @@
 <alg-section-paragraph icon="fa fa-user-plus" label="Invite users" [collapsible]="true">
   <span class="content-title">Add users name list, separated by commas.</span>
   <div class="invite-users">
-    <alg-textarea icon="show" placeholder="login_1, login_2..." (textChange)="onTextChange($event)" [disabled]="state === 'loading'">
+    <alg-textarea [parentForm]="inviteForm" inputName="invitations" icon="show" placeholder="login_1, login_2..." (textChange)="onTextChange($event)">
```

Comment:
why has the "disabled" disappeared?

Reply:
Ok, maybe handled below. I'll see when the other changes will be fixed.

Reply:
> It looks like you're using the disabled attribute with a reactive form directive. If you set disabled to true when you set up this control in your component class, the disabled attribute will actually be set in the DOM for you. We recommend using this approach to avoid 'changed after checked' errors.
>
> Example: 
> form = new FormGroup({
   >  first: new FormControl({value: 'Nancy', disabled: true}, Validators.required),
   >  last: new FormControl('Drew', Validators.required)
>  });

The way disabled was handled worked but threw this warning in the console when used with a reactive form. So I implemented the change the warning suggested:

`inviteForm = this.formBuilder.group({ invitations: { value: '', disabled: this.state == 'loading' } });`

Reply:
`this.state == 'loading'` is evaluated at object initialization, it is not a function.

Reply:
So probably it is a good idea, but the way it is done will not work IMO

Reply:
I've changed the way it's done !